core/src/km_kbp_processevent_api.cpp

```diff
@@ -28,7 +28,7 @@ km_kbp_event(
   switch(event) {
     case KM_KBP_EVENT_KEYBOARD_ACTIVATED:
       assert(data == nullptr);
-      if(data == nullptr) {
+      if(data != nullptr) {
```

Comment:
What's happened here???

```suggestion
      if(data == nullptr) {
```

Reply:
It was incorrect. Look at line 30. Line 31 will only be reached if NDEBUG is defined.

Reply:
Yeah, easy to do, assert(true_condition) followed by if(false_condition) ... almost want a pattern of:

```
assert(!(data != nullptr));
if(data != nullptr) {
  ...
}
```